src/main/java/org/gridsuite/modification/server/dto/CurrentLimitsInfos.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
+ */
+
+@SuperBuilder
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString(callSuper = true)
+@Schema(description = "Current Limits")
+public class CurrentLimitsInfos {
+
+    @Schema(description = "Permanent current limit")
+    private Double permanentCurrentLimit;
```

Comment:
permanentLimit ? (no use to repeat current)

Reply:
ok